src/no-theater-available.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  LitElement,
+  customElement,
+  property,
+  html,
+  TemplateResult,
+  PropertyValues,
+  CSSResult,
+  css,
+} from 'lit-element';
+
+@customElement('ia-no-theater-available')
+export class IANoTheaterAvailable extends LitElement {
+  @property({ type: String }) identifier?: string = '';
+
+  emitLoaded(): void {
+    this.dispatchEvent(
+      new CustomEvent<{ loaded: boolean }>('loadingStateUpdated', {
```

Comment:
Q: What does this event power upstream?

Not a blocker, but was just curious if an `await element.updateComplete` would do the same thing.

Reply:
thanks for the insightful question, so  `loaded` switches the theater between the `<custom-theater-content>` and the `<theater-loader>` that is controlled by our shell, item-nav.

idea is that `<custom-theater-content>` can draw the loader curtain when needed. bookreader isn't doing that right now, but it will once we dynamically load multiple books